### DISCUSSION TITLE
New Mode for Launch Freefall - Launch & drop

### DISCUSSION THREAD

**Discussion by trigger-segfault** [OpenRCT2/OpenRCT2#15644] [Feature requests]
I was watching a video of [The Scream at Six Flags](https://www.youtube.com/watch?v=iVjbTHZMnTU) and I noticed that the ride will launch you once,  launch a second time half-way up, then go all the way to the top then drop you. Once OpenRCT2 is far enough along this would be a great additional mode.

**Comment by Dracheron** [comment]
While we are talking about launched freefall modes - I would also like a mode that would allow you to determine the order and sequence of the drops/launches (and add a way to hold the thing on drop mode if you only want it to drop only a certain height, you could also add in sequence drop from one above station)... hope you follow what I am saying...
